### DISCUSSION TITLE
Terminal is unreadable

### DISCUSSION THREAD

**Discussion by bzigus** [coder/code-server#3978] [Q&A] [answered]
Hey, I'm using Node JS and the terminal is all funky. Got any idea how to fix? 
![image](https://user-images.githubusercontent.com/22013942/129291329-debd1f40-604c-475c-8b52-b14179e8b2bf.png)

**Comment by bzigus** [comment, marked as answer]
Nevermind, seems to be working now

**Reply by jsjoeio** [reply]
What did you do to fix it?

**Reply by bzigus** [reply]
Reloading the page a 1-2 times
